src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -27,12 +27,12 @@ export function DashboardListingController($injector, $scope) {
   const sortableProperties = new SortableProperties([
     {
       name: 'title',
-      getValue: item => item.title,
+      getValue: item => item.title.toLowerCase(),
       isAscending: true,
     },
     {
       name: 'description',
-      getValue: item => item.description,
+      getValue: item => item.description.toLowerCase(),
```

Comment:
I just wanted to make sure. I don't think there is a need to change it as we don't need to guard against behavior that comes from our own code :)

Hopefully we will have functional testing in place before we would ever make a breaking change in the interface of the saved object. I'm not worried about it.